src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1114,6 +1114,11 @@ public sealed class ValidatePatternAttribute : ValidateEnumeratedArgumentsAttrib
         public RegexOptions Options { set; get; } = RegexOptions.IgnoreCase;
 
         /// <summary>
+        /// Gets or sets the custom error message that is displayed to the user
+        /// </summary>
+        public string ErrorMessage { get; set; }
```

Comment:
I don't see any arguments against placement ` ErrorMessage` in `ValidateArgumentsAttribute`.
But I believe that this is bad for UX to add the property in some attributes and does not add to other. 
Why the user can't make a customized message in `ValidateRange`? Ex.: "Year of birth cannot be more 2016".
@lzybkr Could you comment please?